app/assets/stylesheets/pdf.scss

```diff
@@ -196,6 +195,25 @@ h1 {
   page-break-before: always;
 }
 
+@page {
+  size: auto;
+  margin: 0.6in 1in 1.5in 1in;
+}
+
+#header {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+#header img {
+  width: 100%;
+}
+
+#main-content {
+  margin-top: 65px;
+}
```

Comment:
Some styles aren't getting picked up when I test this locally and the image also doesn't seem to be coming through. I'm not sure if this is unique to my setup, maybe we can check in about it on Monday.

![pdf](https://user-images.githubusercontent.com/1887071/38762002-4beb2fc0-3f3c-11e8-90c1-117485e6bc77.png)

Reply:
I pushed a change to inline the stylesheet and use a data URI for the header image, which should be more reliable.